source/gmx/2020.2/CMakeLists.txt.patch

```diff
@@ -0,0 +1,29 @@
+--- /data1/anguse/yingze/gromacs-2020.2/CMakeLists.txt	2021-09-16 00:25:17.125620740 +0800
++++ CMakeLists.txt	2021-09-16 14:54:13.434223510 +0800
+@@ -134,6 +134,26 @@
+ #    (i.e., something that is exposed in installed headers).
+ set(GMX_PUBLIC_LIBRARIES "")
+ 
++# DeepMD
++message(STATUS "Compling with DeepMD...")
++add_definitions(-w) # close warning
++include_directories(${GMX_DEEPMD_ROOT}/include)
++include_directories(${GMX_TENSORFLOW_ROOT}/include)
++link_directories(${GMX_DEEPMD_ROOT}/lib)
++link_directories(${GMX_TENSORFLOW_ROOT}/lib)
++# define high precision
++add_definitions("-D HIGH_PREC")
++# ensure all dynamic libraries linked
++set (CMAKE_EXE_LINKER_FLAGS "-Wl,--no-as-needed")
++# add link libraries
++list (APPEND GMX_PUBLIC_LIBRARIES deepmd_op)
++list (APPEND GMX_PUBLIC_LIBRARIES deepmd_op_cuda)
++list (APPEND GMX_PUBLIC_LIBRARIES deepmd)
++list (APPEND GMX_PUBLIC_LIBRARIES deepmd_cc)
++list (APPEND GMX_PUBLIC_LIBRARIES tensorflow_cc)
++list (APPEND GMX_PUBLIC_LIBRARIES tensorflow_framework)
```

Comment:
`deepmd_cc` should be enough, as others are now a dependency of `deepmd_cc`. (use `ldd` to check it)

Reply:
Also note that `deepmd_op_cuda` is not available in cpu or hip mode.

Reply:
> `deepmd_cc` should be enough, as others are now a dependency of `deepmd_cc`. (use `ldd` to check it)

Unfortunately, according to my test, all libs except `deepmd_op_cuda` should be linked, otherwise the program will crash...

Reply:
Are you using the latest libraries?

Reply:
This is resolved after using the latest version of deepmd-kit